.travis.yml

```diff
@@ -18,6 +18,6 @@ jobs:
   include:
     - php: 5.6
       before_script:
-        - composer install php-parallel-lint/php-parallel-lint
+        - composer update php-parallel-lint/php-parallel-lint
```

Comment:
čekal bych, že install bude fungovat, když to nemá lockfile...

Reply:
taky jsem to čekal, ale bohužel... mimo to nejde nainstalovat celý composer protože phpstan vyžaduje php:^7.0

Reply:
jinak mě to nenapadlo aby to nefailovalo